pkgs/tools/security/diceware/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib
+, pythonPackages
+}:
+
+with pythonPackages;
+
+buildPythonApplication rec {
+  pname = "diceware";
+  version = "0.9.3";
+
+  doCheck = false;
```

Comment:
Why? Add a comment

Reply:
See one of the PR comments above.

Reply:
If you set `doCheck = false`, you should always add a comment inline that specifies the reason